ArduPlane/system.cpp

```diff
@@ -787,11 +770,12 @@ int8_t Plane::throttle_percentage(void)
     }
     // to get the real throttle we need to use norm_output() which
     // returns a number from -1 to 1.
+    float throttle = SRV_Channels::get_output_norm(SRV_Channel::k_throttle);
     if (aparm.throttle_min >= 0) {
-        return constrain_int16(50*(channel_throttle->norm_output()+1), 0, 100);
+        return constrain_int16(100*throttle, 0, 100);
```

Comment:
slightly different logic here.   
was previously -1<T<1 ; 50*(T+1) ;   which results in a T value between 0-2, then multiplied by 50. 
and is now: -1<T<1 ; 100*T  ;    which results in a T value from -1 to 1, then multiplied by 100. 
both have an upper limit of 100, but the old calculation had a lower limit of zero and the new one has a lower limit of -100.